apps/frontend/src/pages/quiz/libs/components/final-questions/libs/constants/final-questions-options.constant.ts

```diff
@@ -0,0 +1,19 @@
+const TASK_DAYS_OPTIONS = [
+	{ label: "Monday", value: 1 },
+	{ label: "Tuesday", value: 2 },
+	{ label: "Wednesday", value: 3 },
+	{ label: "Thursday", value: 4 },
+	{ label: "Friday", value: 5 },
+	{ label: "Saturday", value: 6 },
+	{ label: "Sunday", value: 7 },
+];
+
+const ALLOW_NOTIFICATIONS_OPTIONS = [
+	{ label: "Yes, I’d love daily motivation!", value: "true" },
```

Comment:
I've opened a PR in vscode.dev, and `'`in `I'd` word is highlighted there. ESLint shows `Parsing error: Invalid character.eslint` when I copied it to my VSCode.
 The `'` might be a non-Latin character.